interface/super/rules/controllers/detail/view/view_summary.php

```diff
@@ -24,84 +24,225 @@
+                                            return xla($data);
+                                        }
+                                        return $data;
+                                    }
+                                    if ($something) {
+                                        if ($rule->hasRuleType(RuleType::from('activealert')) || $rule->hasRuleType(RuleType::from('passivealert'))) {
+                                            $clinical = '1';
+                                        }
+                                        if ($rule->hasRuleType(RuleType::from('activealert')) && $rule->hasRuleType(RuleType::from('passivealert'))) {
+                                            $timing .= "This CR has both an
+                                                            <span class='bold'
+                                                                  data-toggle='popover'
+                                                                  data-trigger='hover'
+                                                                  data-placement='auto'
+                                                                  title='Active Alerts'
+                                                                  data-content='A Pop-up will occur daily when the demographics page is opened listing any Treatment Goals needing attention.'>Active Alert
+                                                             </span>
+                                                             and a
+                                                            <span class='bold'
+                                                                  data-toggle='popover'
+                                                                  data-toggle='popover'
+                                                                  data-trigger='hover'
+                                                                  data-placement='auto'
+                                                                  title='Passive Alerts'
+                                                                  data-content='These alerts appear on the Dashboard page inside the CR widget'>Passive Alert</span>.
+                                                            <br /> ".text($timings['clinical']['pre']['amount']). " " .text($timings['clinical']['pre']['timeUnit2']). "
+                                                            before its Due date, this CR is marked <span class='due_soon bolder'>Due Soon</span>. <br />";
+                                            $timing .= "Then for ".text($timings['clinical']['post']['amount'])." ".$timings['clinical']['post']['timeUnit2']." it is <span class='due_now'>Due</span>. ";
+                                            $timing .= "After this, it is marked as <span class='past_due'>Past due</span>.";
+                                            $timing = xlts($timing);
```

Comment:
just pointing out another flaw with these xlts functions. Basically impossible to collect those and send them to the translation spreadsheet.